Gemfile

```diff
@@ -103,6 +103,7 @@ group :development, :test do
   gem 'fcrepo_wrapper'
   gem 'rspec-rails'
   gem 'dotenv-rails'
+  gem 'xray-rails'
```

Comment:
This should move down to under the `group: development` on 109.  This adds to dev and test, which means extra overhead on Travis tests.